go.mod

```diff
@@ -49,7 +49,7 @@ require (
 	github.com/moby/sys/signal v0.7.0
 	github.com/morikuni/aec v1.0.0
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.1.0-rc2.0.20221005185240-3a7f492d3f1b
+	github.com/opencontainers/image-spec v1.1.0-rc3
```

Comment:
Note that the version that was used (https://github.com/opencontainers/image-spec/commit/3a7f492d3f1b) already contains the fixes for [CVE-2021-41190](https://github.com/advisories/GHSA-mc8v-mgrf-8f4m);

- https://github.com/opencontainers/image-spec/commit/2eb4046685179aa5392bd938a21a20cd721fc133
- https://github.com/opencontainers/image-spec/commit/5b82148fa647734a2c6095b2eab9b502fbca227e

Reply:
Yes, understood. This was updated transitively by pulling in `github.com/docker/distribution@v2.8.2-beta.1` so I didn't think it was a good possible nor a good idea to try downgrade it.